content/posts/weekly/June-2023.md

```diff
@@ -58,3 +58,15 @@ weight: 51
 
 - If you are interested about the manual triggering deployed two weeks ago, David Kornel and Jakub Stejskal published [a blog post](https://fedoramagazine.org/how-to-use-testing-farm-outside-of-rhel/) about this on fedoramagazine.org.
 - Copr builds configured as a monorepo job will now be built in one Copr project together. The bug with overwriting the statuses for monorepo jobs has been fixed as well. ([packit-service#2097](https://github.com/packit/packit-service/pull/2097))
+
+## Week 26 (June 27th – July 3rd)
+
+- The bug which prevented errors on Testing Farm request submission from being
+  correctly propagated to the user was fixed.
+  ([packit-service#2103](https://github.com/packit/packit-service/pull/2103))
+- `pull_from_upstream` jobs can now be retriggered with a comment
+  `/packit pull-from-upstream` in a dist-git pull request.
+  ([packit-service#2087](https://github.com/packit/packit-service/pull/2087))
+- Copr builds configured as a monorepo job will now be built in one Copr project together.
+  The bug with overwriting the statuses for monorepo jobs was fixed as well.
+  ([packit-service#2097](https://github.com/packit/packit-service/pull/2097))
```

Comment:
```suggestion
```

Reply:
this one was already included last week

Reply:
That's because I've moved the branches on Wednesday...